ERCS/erc-7624.md

```diff
@@ -0,0 +1,86 @@
+---
+eip: 7624
+title: Dynamic Identity Binding Soulbound Tokens
+description: Dynamic identity binding for SBT to provide a more resilient and user-centric solution
+author: Auph (@auph)
+discussions-to: https://ethereum-magicians.org/t/erc-7624-dynamic-identity-binding-soulbound-tokens/18676
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-10-18
+requires: 721
+---
+
+## Abstract
+
+This Ethereum Improvement Proposal introduces a mechanism for dynamic identity binding to enhance the security and continuity of users' identities in the context of Soulbound tokens. The proposal addresses the current limitation where Soulbound tokens are static and irrevocable, making it challenging for users to recover their identity in case of wallet compromise. The proposed solution allows users to re-identify themselves with a new wallet address while preserving the integrity of their Soulbound identity.
```

Comment:
```suggestion
This EIP introduces a mechanism for dynamic identity binding to enhance the security and continuity of users' identities in the context of Soulbound tokens. The proposal addresses the current limitation where Soulbound tokens are static and irrevocable, making it challenging for users to recover their identity in case of wallet compromise. The proposed solution allows users to re-identify themselves with a new wallet address while preserving the integrity of their Soulbound identity.
```